tests/pdarray_creation_test.py

```diff
@@ -50,10 +50,19 @@ def testArrayCreation(self):
             ak.array(list(list(0)))
 
     def test_arange(self):
-        self.assertTrue((ak.array([0, 1, 2, 3, 4]) == ak.arange(0, 5, 1)).all())
-        self.assertTrue((ak.array([5, 4, 3, 2, 1]) == ak.arange(5, 0, -1)).all())
-        self.assertTrue((ak.array([-5, -6, -7, -8, -9]) == ak.arange(-5, -10, -1)).all())
-        self.assertTrue((ak.array([0, 2, 4, 6, 8]) == ak.arange(0, 10, 2)).all())
+        self.assertListEqual(
+            ak.array([0, 1, 2, 3, 4]).to_ndarray().tolist(), ak.arange(0, 5, 1).to_ndarray().tolist()
```

Comment:
A lot of places in this file that the ak.array could just be defined as a list instead